### DISCUSSION TITLE
I wrote some documentation and examples about networking and rootless Podman

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#18945] [Show and tell]
See https://github.com/eriksjolund/podman-networking-docs

If you think it makes sense to add or merge this documentation somewhere under https://github.com/containers/podman please let me know. I could create a PR. In any case, it's probably good to let the text mature a bit. First some time is needed to fix errors and such.

**Comment by Luap99** [comment]
Thanks, some feedback from quick look.

The `Source address preserved` table contains the same info as the table directly above so it fells redundant.

Also it may be good to provide some info about using named networks as rootless, i.e. `podman network create`. Currently this mode uses slirp4netns only but I am working on enabling pasta there as well.
You can see here how this works: https://containers.github.io/podman.io_old/old/community/meeting/notes/2021-10-05/Podman-Rootless-Networking.pdf, check the Rootless CNI section. Note that this works the same for netavark now since 4.0.

**Comment by eriksjolund** [comment]
Thanks for feedback! (Sorry for late reply).
I removed the `Source address preserved` table and added some sections about network backends
https://github.com/eriksjolund/podman-networking-docs#network-backends
Those sections do not contain much information right now but I added the curl/nginx example from
https://containers.github.io/podman.io_old/old/community/meeting/notes/2021-10-05/Podman-Rootless-Networking.pdf
with some modifications. I noticed that the  `--rootless-netns` could be used instead of `--rootless-cni`.

**Comment by techcomplit** [comment]
Hello, everyone. Hey, i’m using docker with podman to consolidate my micro-services architecture in REHL8 (as REHL8 doesn’t have direct compatibility with docker) Everything is working but the NATS communication between two containers via the 4222 port, do not know why. Any top-of-mind idea of what could be happening or if this is even possible given docker + podman setup? I can provide with additional information, of course.

**Reply by eriksjolund** [reply]
@techcomplit To create a new discussion thread, go to https://github.com/containers/podman/discussions/new/choose
and click _Get started_ next to _General_ or _Q & A_.
